keps/sig-storage/3751-volume-attributes-class/README.md

```diff
@@ -172,27 +172,48 @@ type PersistentVolumeClaimSpec struct {
   ...
 }
 
+// Add the PersistentVolumeClaimModifyVolume condition to existing PersistentVolumeClaimConditionType
+// Condition is used to document the last error controller sees
+const (
+  ...
+  // PersistentVolumeClaimVolumeAttriburesModifyError - a user trigger modify volume of pvc has error
+  PersistentVolumeClaimVolumeAttriburesModifyError PersistentVolumeClaimConditionType = "VolumeAttriburesModifyError"
+  ...
+)
+
+
 // PersistentVolumeClaimStatus represents the status of PV claim
 type PersistentVolumeClaimStatus struct {
   ...
-  // The VolumeAttributesClassName string cannot be empty
-  VolumeAttributesClassName string
-  ModifyVolumeStatus *PersistentVolumeClaimModifyVolumeStatus
+  // ModifyVolumeStatus represents the status object of ControllerModifyVolume operation
+  ModifyVolumeStatus ModifyVolumeStatus
   ...
 }
 
+// ModifyVolumeStatus represents the status object of ControllerModifyVolume operation
+type ModifyVolumeStatus struct {
+  // CurrentVolumeAttributesClassName is the current name of the VolumeAttributesClass the PVC is using
+  CurrentVolumeAttributesClassName string
+  // TargetVolumeAttributesClassName is the name of the VolumeAttributesClass the PVC currently being reconciled
+  TargetVolumeAttributesClassName string
+  // Status is the status of the ControllerModifyVolume operation
+  Status *PersistentVolumeClaimModifyVolumeStatus
+}
+
 // +enum
+// New statuses can be added in the future. Consumers should check for unknown statuses and fail appropriately
 type PersistentVolumeClaimModifyVolumeStatus string
 
 const (
-  // When modify volume is complete, the empty string is set by modify volume controller.
-  PersistentVolumeClaimNoModifyVolumeInProgress PersistentVolumeClaimModifyVolumeStatus = ""
-  // State set when modify volume controller starts modifying the volume in control-plane
+  // Pending indicates that the PersistentVolumeClaim cannot be modified due to requirements not being met, such as 
+  // the PersistentVolumeClaim being in an invalid state or the specified VolumeAttributesClass is existing
```

Comment:
```suggestion
  // the PersistentVolumeClaim being in an invalid state or the specified VolumeAttributesClass doesn't exist.
```

Reply:
Oh actually "the VolumeAttributesClass doesn't exist should be moved to pending state instead".